commit c37bab0bab803a6480c4e2259a39676fec9244ed
Author: Matt Flax <flatmax@flatmax.org>
Date:   Mon Mar 27 22:26:49 2017 +1100

    Audioinjector addons dts : add reset and regulators for the Octo product.
    
    This patch adds a reset GPIO for the AudioInjector.net octo sound card.
    This patch adds missing regulators for the cs42448 codec.

diff --git a/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts b/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts
index dbf2f3cacc2e..a36fa85a61f7 100644
--- a/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts
+++ b/arch/arm/boot/dts/overlays/audioinjector-addons-overlay.dts
@@ -13,6 +13,26 @@
 	};
 
 	fragment@1 {
+		target = <&soc>;
+			__overlay__ {
+				reg_digital: reg_digital@0 {
+				  compatible = "regulator-fixed";
+				  regulator-name = "cs42448_dig_supply";
+				  regulator-min-microvolt = <3300000>;
+				  regulator-max-microvolt = <3300000>;
+				  regulator-always-on;
+  			};
+				reg_analogue: reg_analogue@0 {
+				  compatible = "regulator-fixed";
+				  regulator-name = "cs42448_ana_supply";
+				  regulator-min-microvolt = <5000000>;
+				  regulator-max-microvolt = <5000000>;
+				  regulator-always-on;
+			};
+		};
+	};
+
+	fragment@2 {
 		target = <&i2c1>;
 		__overlay__ {
 			#address-cells = <1>;
@@ -25,6 +45,10 @@
 				reg = <0x48>;
 				clocks = <&cs42448_mclk>;
 				clock-names = "mclk";
+				VA-supply = <&reg_analogue>;
+				VD-supply = <&reg_digital>;
+				VLS-supply = <&reg_digital>;
+				VLC-supply = <&reg_digital>;
 				status = "okay";
 			};
 
@@ -36,12 +60,13 @@
 		};
 	};
 
-	fragment@2 {
+	fragment@3 {
 		target = <&sound>;
 		__overlay__ {
 			compatible = "ai,audioinjector-octo-soundcard";
 			mult-gpios = <&gpio 27 0>, <&gpio 22 0>, <&gpio 23 0>,
 				     <&gpio 24 0>;
+			reset-gpios = <&gpio 5 0>;
 			i2s-controller = <&i2s>;
 			codec = <&cs42448>;
 			status = "okay";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1462_c37bab0bab803a6480c4e2259a39676fec9244ed.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1462_c37bab0bab803a6480c4e2259a39676fec9244ed.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
